utils/build-script-impl

```diff
@@ -805,15 +814,20 @@ function is_cross_tools_deployment_target() {
 # A list of deployment targets that we compile or cross-compile the
 # Swift standard library for.
 STDLIB_DEPLOYMENT_TARGETS=()
-
-case "$(uname -s)" in
-    Linux)
+# FIXME: This will not work while trying to cross-compile
+# It gets the architecture from the system processor arch.
```

Comment:
I think this one can be removed as well.
As long as I am understanding it correctly: